app/Http/Livewire/StreamList.php

```diff
@@ -13,8 +13,19 @@ class StreamList extends Component
 {
     use WithPagination;
 
+    protected $queryString = [
+        'search' => ['except' => ''],
+    ];
+
     public bool $isArchive = false;
 
+    public ?string $search = null;
+
+    public function updatedSearch($value): void
```

Comment:
I know it is not being used, but can you still typehint it and call it `search`

Reply:
We cannot cal it `search` because it's a livecycle hook from Livewire (https://www.laravel-livewire.com/docs/2.x/lifecycle-hooks). I use it to reset the page number to page 1 when searching. (We we don't do that and enter a search query when we are on page 10 for example it will remain on page 10 and we don't see any results). 

I will look into the typehint.

Reply:
Sry I wasn't clear. I was only talking about the `$value` variable.

Reply:
A. We could actually remove the variable because it's not used.